packages/paste-website/src/components/empty-state/WatchThisSpace.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Anchor} from '@twilio-paste/anchor';
+import {Flex} from '@twilio-paste/flex';
+
+import {StaticImage} from 'gatsby-plugin-image';
+
+export const WatchThisSpace: React.FC = () => {
+  return (
+    <Card>
+      <Flex vAlignContent="center">
+        <Box marginRight="space70">
+          <StaticImage
+            src="../../assets/images/patterns/empty-awaiting-data.png"
+            alt="This content has not been published"
```

Comment:
Suggestion: this image is decorative because all the info they need is in text below. You can just have `alt=""`.